internal/core/graphics/brush.rs

```diff
@@ -125,6 +125,49 @@ impl Brush {
             )),
         }
     }
+
+    /// Returns a new version of this brush with the opacity changed by _subtracting_
+    /// `amount` from the opacity for all related colors. The reference is the opacity's
+    /// normalized value as `f32`.
+    ///
+    /// This means that a _positive_ `amount` will yield a more transparent brush and
+    /// a _negative_ `amount` will yield a more opaque brush.
+    #[must_use]
+    pub fn faded(&self, amount: f32) -> Self {
```

Comment:
I added `opaque` in a later commit to be used instead of that `#1238.translucent(-30%)` case. But the usage would actually be `color.opaque(1.3)` (in Rust)...